### PR TITLE
Fix: Usability improvements

### PR DESCRIPTION
**Pull request by tiemobang**
This pull request contributes a number of fixes to catch incorrect file paths provided by the user.

Further it extends the dashboard with handling of missing values (NULL values).

Also it introduces variables for the configuration files to refer to common paths: the variable ${LST_BENCH_HOME} for the root directory of the project and the variables ${LIB_PATH}, ${WL_PATH}, ${EXP_PATH}, ${TEL_PATH} to refer to the file path of the configuration files of the library, workload, experiment, and telemetry.

### CODE REVIEW

**Comment by tiemobang** [comment]
@microsoft-github-policy-service agree

**Comment by jcamachor** [comment]
Thanks for your contribution, @tiemobang ! Somehow I missed this PR, I'll take a look shortly.

File: metrics/app/main.py [hunk 1]
```diff
@@ -23,33 +25,49 @@
 
 
 @st.cache_resource
-def get_connection():
+def get_connection(*, result_dirs: List[str] = None):
+    # Either search for results in provided direcotries
```
Comment (jcamachor) [inline metrics/app/main.py:29]: typo: direcotries -> directories

File: launcher.sh [hunk 2]
```diff
@@ -1,7 +1,11 @@
 #!/bin/bash -e
 
 # Constants
-LST_BENCH_HOME="$PWD"
+# Directory of the script
+export LST_BENCH_HOME="$(dirname "$(readlink -f "$0")")"
 LST_BENCH_CLASSPATH="$LST_BENCH_HOME/target/*:$LST_BENCH_HOME/target/lib/*:$LST_BENCH_HOME/target/classes/*"
 
-java -cp $LST_BENCH_CLASSPATH com.microsoft.lst_bench.Driver "$@"
+SPARK_CLASSPATH="${SPARK_HOME}/jars/*"
```
Comment (jcamachor) [inline launcher.sh:8]: I'm wondering why is the Spark class path included here, i.e., launcher does not set up the engine for tests?

File: metrics/app/README.md [hunk 3]
```diff
@@ -36,6 +36,8 @@ To include data from a new system, duplicate one of the directories in the [run
 For a deeper understanding of the directory structure, consult the [README file](/run/README.md). 
 The LST-Bench dashboard web app automatically retrieves results from the .duckdb files within those folders and displays them on the dashboard.
 
+Alternatively, you can provide your own paths to search for results via commandline arguments, see below.
```
Comment (jcamachor) [inline metrics/app/README.md:39]: Nice!

**Review by jcamachor — CHANGES_REQUESTED** [review]
Just a couple of minor comments.

**Comment by jcamachor** [comment]
Fixed in [364e21b](https://github.com/microsoft/lst-bench/commit/364e21b223d40cff7ec3ba5d6449a117f061f291) . Thanks for your contribution, @tiemobang !

**Pull request closed without merging**